tests/integ/modin/series/test_align.py

```diff
@@ -94,19 +71,30 @@ def test_align_basic_series_reorder_index(join, axis):
         ],
         index=["G", "H", "M", "A"],
     )
-    native_left, native_right = native_ser.align(
-        native_other_ser,
-        join=join,
-        axis=axis,
-        limit=None,
-        fill_axis=0,
-        broadcast_axis=None,
-    )
-    ser = pd.Series(native_ser)
-    other_ser = pd.Series(native_other_ser)
-    left, right = ser.align(other_ser, join=join, axis=axis)
-    assert_snowpark_pandas_equals_to_pandas_without_dtypecheck(left, native_left)
-    assert_snowpark_pandas_equals_to_pandas_without_dtypecheck(right, native_right)
+    eval_series_align_result(native_ser, native_other_ser, join=join, axis=axis)
+
+
+@pytest.mark.parametrize("join", ["outer", "inner", "left", "right"])
```

Comment:
I'm not clear if we support `right` or not. The docstring in `ordered_dataframe.py` does not mention it.

Reply:
that is for align api, not the ordered dataframe, it seems supported right now. i am also not very sure how it is supported @sfc-gh-lmukhopadhyay